mllib/src/main/scala/org/apache/spark/mllib/clustering/GaussianMixtureModel.scala

```diff
@@ -83,5 +95,82 @@ class GaussianMixtureModel(
       p(i) /= pSum
     }
     p
-  }  
+  }
+}
+
+@Experimental
+object GaussianMixtureModel extends Loader[GaussianMixtureModel] {
+
+  private object SaveLoadV1_0 {
+
+    case class Data(weights: Array[Double], mus: Array[Vector], sigmas: Array[Matrix])
```

Comment:
As I mentioned before, let's flatten the data into rows, where each row corresponds to a Gaussian distribution.